lib/config.js

```diff
@@ -129,6 +133,10 @@ Config.prototype.buildArgs = function () {
     brave_referrals_api_key: this.braveReferralsApiKey,
     enable_hangout_services_extension: this.enable_hangout_services_extension,
     enable_cdm_host_verification: this.brave_enable_cdm_host_verification,
+    brave_android_keystore_path: this.androidKeystorePath,
```

Comment:
see https://github.com/brave/brave-core/pull/4233/files#r358508527

Reply:
Also please make sure all the names are consistent. If we have `brave_android_key_password` it should map to `this.braveAndroidKeyPassword` and `getNPMConfig(['brave_android_key_password'])`

Reply:
Changed the names of variables